script.js

```diff
@@ -0,0 +1,231 @@
+
+class Player {
+	constructor(name, symbol) {
+		this.name = name,
+		this.symbol = symbol
+	}
+}
+
+const Board = ( () => {
+	const boardArray = [null, null, null, null, null, null, null, null, null];
+	const winCombinations = [
+					[0,1,2],
+					[3,4,5],
+					[6,7,8],
+					[0,3,6],
+					[1,4,7],
+					[2,5,8],
+					[0,4,8],
+					[6,4,2]
+					];
+    
+	function render() {
```

Comment:
Let's not mix DOM related logic with our `Board` factory. Our board factory's methods should only include:

1. A function that updates the `boardArray` depending on player's choice
2. A function that checks whether a player's current set of moves already is within any array in  `winCombinations`
3. A function that checks whether or not `boardArray` is full (checking if none of the elements are null).